src/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx

```diff
@@ -67,16 +68,22 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
       ZeroState={ZeroState}
       userPreferredMetric={userPreferences?.metric}
     >
-      <ArtworkFilterSavedSearchAlertContextProvider entity={savedSearchEntity}>
-        <BaseArtworkFilter
-          relay={relay}
-          viewer={artist}
-          Filters={<ArtistArtworkFilters />}
-          relayVariables={{
-            aggregations: ["TOTAL"],
-          }}
-        />
-      </ArtworkFilterSavedSearchAlertContextProvider>
+      <ArtworkFilterAlertContextProvider
```

Comment:
Curious why everything needs to be wrapped all around the app, vs placing it once in BaseArtworkFilter, or wrapping a component above that (but yet internal to it)